oracle/src/oracleimporter.h

```diff
@@ -7,11 +7,35 @@
 #include <utility>
 
 // many users prefer not to see these sets with non english arts
-// as a solution we remove the date property on these sets
-// that way they will be sorted last by default
-// this will cause their art to not get priority over english cards
-// users will still be able to find these sets and prioritize them manually
 const QStringList nonEnglishSets = {"4BB", "FBB", "PS11", "PSAL", "REN", "RIN"};
+const QMap<QString, CardSet::Priority> setTypePriorities{
+    {"core", CardSet::PriorityPrimary},
+    {"expansion", CardSet::PriorityPrimary},
+
+    {"commander", CardSet::PrioritySecondary},
+    {"starter", CardSet::PrioritySecondary},
+
+    {"archenemy", CardSet::PriorityReprint},
+    {"arsenal", CardSet::PriorityReprint},
+    {"box", CardSet::PriorityReprint},
+    {"draft_innovation", CardSet::PriorityReprint},
```

Comment:
modern horizons and jumpstart are counted as draft innovations, they contain new cards that preferably get a higher priority than promo sets

this also applies but less so to duel decks, which only rarely include new cards.

Reply:
Good catch. I moved those sets to the `PrioritySecondary` category. I thought about making a `PriorityTertiary` classification for them, but it didn't seem necessary.